signer/src/storage/in_memory.rs

```diff
@@ -927,6 +927,69 @@ impl super::DbRead for SharedStore {
             .map(|s| (s.will_sign, s.aggregate_key)))
     }
 
+    // This might look like it do not have same behaviour as the implementation from postgres.rs
+    // However, this is correct, e use the stacks block anchor block as a proxy for the timestamp of
+    // when the decision was made.
```

Comment:
nit: I would use something like this
```suggestion
    // The postgres implementation uses a timestamp to figure out when a
    // decision was inserted into the database. The in memory database
    // does not have such a timestamp, so we use the stacks block anchor
    // block as a proxy for when the decision was made.
```